src/style.css

```
@@ -1,3 +1,102 @@
-body {
-  margin: 0;
+html {
+  box-sizing: border-box;
 }
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+  }
+
+  body {
+    margin: 0;
+    font-family: Roboto, sans-serif;
+    font-style: normal;
+    font-weight: normal;
+  }
+
+  a {
+    text-transform: uppercase;
+  }
+
+  /* Header start */
+
+  active-color {
```

Need to create variable with `:root` selector
```suggestion
  :root {
```